ui/apps/platform/src/Containers/Policies/Wizard/Step2/PolicyBehaviorForm.tsx

```diff
@@ -234,6 +248,7 @@ function PolicyBehaviorForm({ hasActiveViolations }: PolicyBehaviorFormProps) {
                         fieldId="policy-event-source"
                         label="Event sources (Runtime lifecycle only)"
                         isRequired={hasRuntime}
+                        helperText={eventSourceHelperText}
```

Comment:
Commenting on a self-documenting alternative to stimulate out-of-the-box thinking in the future.

Vertical layout and include helper text in the option label.
* Deployment: event sources that include process and network activity, pod exec, and pod port forwarding.
* Audit logs: event sources that match Kubernetes audit log records